flocker/testtools/__init__.py

```diff
@@ -215,12 +215,16 @@ def loop(result):
     return d
 
 
-def random_name():
-    """Return a short, random name.
+def random_name(case):
+    """
+    Return a short, random name.
```

Comment:
Not as short as it used to be...pretty long in fact.
I had a feeling that I tried to do something like this before and that there was a restriction on the length of Docker image and container names....but perhaps that was a geard restriction.
I'll quickly search.